utils/ClientDevicesLib/sqlite.go

```diff
@@ -0,0 +1,136 @@
+package client_devices
+
+import (
+	"database/sql"
+
+	log "github.com/sirupsen/logrus"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+//SqlLiteDevicesLib implements the ClientDeviceLib interface
+type SqlLiteDevicesLib struct {
+	db *sql.DB
+}
+
+func init_db(db *sql.DB) error {
+	stmt, _ := db.Prepare("create table if not exists client_devices( " +
+		" name text, platform text, mac_address text, ip_address varchar(15), stats_table text);")
+	_, err := stmt.Exec()
+	return err
+}
+
+// NewClient will initialize the Database and return the client
+func NewClient() (*SqlLiteDevicesLib, error) {
```

Comment:
Name should indicate that it's using sql. Otherwise from outside of the package it's not obvious.

Can't think of a great reason for this so it's probably fine as is, but what I usually see is the return type being the interface it implements.
Only benefit I can think of is compilation will fail here if the interface changes, vs the way you have it now compilation will fail only when someone tries to assign it to a ClientDevicesLib var.